seeker_test.go

```diff
@@ -26,15 +26,16 @@ func TestSeeker(t *testing.T) {
 	f.WriteString("Line 1\n")
 	f.WriteString("Line 2\n")
 
-	s := NewSeeker(f)
+	s := New(f)
 
 	if err = s.SeekToStart(); err != nil {
 		t.Fatal(err)
 	}
 
 	var str string
-	if err = s.ReadLine(func(buf *bytes.Buffer) {
+	if err = s.ReadLine(func(buf *bytes.Buffer) error {
```

Comment:
Hmm all the funcs you pass into ReadLine never return an err..always nil..is it needed?